ui/src/containers/Layout/style.module.scss

```diff
@@ -1,59 +1,106 @@
 @import 'src/scss/lib';
 
 .container {
+  display: flex;
+  height: 100vh;
+  position: relative;
+  width: 100vw;
+}
+
+.sidebar {
   align-items: center;
-  margin: 0 auto;
-  max-width: 1024px;
-  padding-top: 66px;
+  border-right: 1px solid $color-button-bg;
```

Comment:
We are using `$color-button-bg` for coloring the border. Please check if there is an appropriate color for boundaries, and if not, please create a new color variable. Reusing variables in unrelated elements will likely end badly (e.g we might change the style of our buttons only to find of sidebar is not purple).